tests/bootstrap.php

```diff
@@ -2,4 +2,14 @@
 
 declare(strict_types=1);
 
-require __DIR__ . '/../vendor/autoload.php';
+# loads the composer class loader
+require_once(__DIR__ . "/../vendor/autoload.php");
+
+# configure session/globals
+@session_start();
+$_SESSION["site_id"] = "default";
+$backpic = "";
+$ignoreAuth = true;
+
+# load globals
+require_once(__DIR__ . "/../interface/globals.php");
```

Comment:
hopefully all you should need for this file is:
```
declare(strict_types=1);
$ignoreAuth = true;
require_once(__DIR__ . "/../interface/globals.php");
```
(I'm gonna try this stuff out this evening and see what I can make explode)

Reply:
actually, think it will need to be:
```
declare(strict_types=1);
$_GET['site'] = 'default';
$ignoreAuth = true;
require_once(__DIR__ . "/../interface/globals.php");
```